packages/next/export/index.js

```diff
@@ -140,7 +140,7 @@ export default async function (dir, options, configuration) {
     distDir,
     buildId
   })
-  exportPathMap['/404.html'] = exportPathMap['/404.html'] || { page: '/_error' }
+  exportPathMap['/404'] = exportPathMap['/404'] || { page: '/_error' }
```

Comment:
```suggestion
  if (!exportPathMap['/404']) {
    exportPathMap['/404.html'] = exportPathMap['/404.html'] || { page: '/_error' }
  }
```

It looks like the `export` test suite is failing, we can just check if there is already a `/404` page and not add the `/404.html` when there is